website/docs/cloud-docs/api-docs/github-app/installations.mdx

```diff
@@ -0,0 +1,121 @@
+---
+page_title: GitHub App Installations - API Docs - Terraform Cloud
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+
+[201]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201
+
+[202]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202
+
+[204]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204
+
+[400]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400
+
+[401]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
+
+[403]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
+
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+
+[409]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
+
+[412]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412
+
+[422]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422
+
+[429]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429
+
+[500]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500
+
+[504]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504
+
+[JSON API document]: /terraform/cloud-docs/api-docs#json-api-documents
+
+[JSON API error object]: https://jsonapi.org/format/#error-objects
+
+# GitHub App Installations API
+
+GitHub App installations are created in Terraform Cloud UI by user interation. [Learn more about GitHub App in Terraform Cloud](/terraform/cloud-docs/vcs/github-app)
+
+-> **Note:** This API is restricted to Terraform Cloud User's who have previously authorized the Terraform Cloud GitHub App. You can manage your [GitHub App token](/terraform/cloud-docs/users-teams-organizations/users#github-app-oauth-token) from **User Settings** > **Tokens**.
+
+## List Installations
+
+This endpoint lists GitHub App installations available to the current User.
+
+`GET /github-app/installations`
+
+### Query Parameters
+
+| Parameter                               | Description                                                                                                                                             |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `filter[name]`                          | **Optional.** If present, returns a list of available GitHub App installations that match the GitHub organization or login.                             |
+| `filter[installation_id]`               | **Optional.** If present, returns a list of available GitHub APp installations that match the installation id within GitHub. (**Not Terraform Cloud**) |
```

Comment:
```suggestion
| `filter[installation_id]`               | **Optional.** If present, returns a list of available GitHub App installations that match the installation id within GitHub. (**Not Terraform Cloud**) |
```

Reply:
I'm also not exactly sure what `Not Terraform Cloud` means in this context. Maybe this is `Terraform Enterprise only`?

Reply:
Ohh we want to clarify the distinctions between `installation_id` which is the GitHub specific ID and the ID we have in Terraform Cloud.